### DISCUSSION TITLE
how to enable sharedarraybuffer

### DISCUSSION THREAD

**Discussion by Icepartition** [cefsharp/CefSharp#3323] [Q&A] [answered]
how to enable sharedarraybuffer

**Comment by amaitland** [comment, marked as answer]
Can you clarify please. Are you seeing an error when you attempt to create a SharedArrayBuffer? 

Possibly disabled by Chromium as site isolation trials are disabled. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer/SharedArrayBuffer
